static/app/utils/fields/index.ts

```diff
@@ -0,0 +1,574 @@
+    desc: 'Determines if a tag or field exists in an event',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.RELEASE]: {
+    desc: 'The version of your code deployed to an environment',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.RELEASE_BUILD]: {
+    desc: 'The full version number that identifies the iteration',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.RELEASE_PACKAGE]: {
+    desc: 'The identifier unique to the project or application',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.RELEASE_STAGE]: {
+    desc: 'Stage of usage (i.e., adopted, replaced, low)',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.RELEASE_VERSION]: {
+    desc: 'An abbreviated version number of the build',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.SDK_NAME]: {
+    desc: 'Name of the platform that sent the event',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.SDK_VERSION]: {
+    desc: 'Version of the platform that sent the event',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_ABS_PATH]: {
+    desc: 'Absolute path to the source file',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_COLNO]: {
+    desc: 'Column number of the call starting at 1',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.NUMBER,
+  },
+  [FieldKey.STACK_FILENAME]: {
+    desc: 'Relative path to the source file from the root directory',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_FUNCTION]: {
+    desc: 'Name of function being called',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_IN_APP]: {
+    desc: 'Indicates if frame is related to relevant code in stack trace',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.BOOLEAN,
+  },
+  [FieldKey.STACK_LINENO]: {
+    desc: 'Line number of the call starting at 1',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.NUMBER,
+  },
+  [FieldKey.STACK_MODULE]: {
+    desc: 'Platform specific module path',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_PACKAGE]: {
+    desc: 'The package the frame is from',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_RESOURCE]: {
+    desc: 'The package the frame is from',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [FieldKey.STACK_STACK_LEVEL]: {
+    desc: 'Number of frames per stacktrace',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.NUMBER,
+  },
+  [FieldKey.TIMES_SEEN]: {
+    desc: 'Total number of events',
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.NUMBER,
```

Comment:
Question: is there any difference between `valueType` NUMBER and STRING? Do we handle those differently in the UI?

Reply:
@malwilley Not yet, the `valueType` in fields aren't used yet outside of aggregations in #36386 but I added them to the definitions so in the future if we want to use them in the UI in some way it would already be defined 😁

Reply:
Okay cool just wanted to check in case I was missing something. Good to think ahead!